cloudbuild.yaml

```diff
@@ -0,0 +1,26 @@
+steps:
```

Comment:
You don't need to build any binaries first?

Reply:
The binaries are built in the Dockerfile, so it's self-contained.